From foo@baz Tue Apr  9 12:12:43 2002
To: Greg KH <greg@kroah.com>
Date: 02 Mar 2005 12:18:19 -08:00
From: macro@linux-mips.org
Subject: [PATCH] I2C: Enable I2C_PIIX4 for 64-bit platforms

 Is there any specific reason for the PIIX4 SMBus driver to be disabled on
64-bit platforms?  If not, then please apply the following change.  The
MIPS Technologies Malta development board has the 82371EB chip and
supports 64-bit configurations.  I've verified the driver to work
correctly using 64-bit kernels for both endiannesses.

Signed-off-by: Maciej W. Rozycki <macro@mips.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -Nru a/drivers/i2c/busses/Kconfig b/drivers/i2c/busses/Kconfig
--- a/drivers/i2c/busses/Kconfig	2005-03-02 16:32:32 -08:00
+++ b/drivers/i2c/busses/Kconfig	2005-03-02 16:32:32 -08:00
@@ -287,7 +287,7 @@
 
 config I2C_PIIX4
 	tristate "Intel PIIX4"
-	depends on I2C && PCI && EXPERIMENTAL && !64BIT
+	depends on I2C && PCI && EXPERIMENTAL
 	help
 	  If you say yes to this option, support will be included for the Intel
 	  PIIX4 family of mainboard I2C interfaces.  Specifically, the following
